### PR TITLE
Add stub for wolfSSL_set_ecdh_auto.

### PR DESCRIPTION
**Pull request by kareem-wolfssl**
# Description

Please describe the scope of the fix or feature addition.

Fixes zd#16927

# Testing

Unit test

# Checklist

 - [X] added tests
 - [ ] updated/added doxygen
 - [ ] updated appropriate READMEs
 - [ ] Updated manual and documentation

### CODE REVIEW

**Comment by kareem-wolfssl** [comment]
Waiting for customer confirmation before merging this.

**Review by dgarske — APPROVED** [review]

**Comment by dgarske** [comment]
Retest this please

**Pull request merged**
